### PR TITLE
DT-190 added dev helm deploy

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: helm_deploy/community-proxy/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ template "app.name" . }}
+    chart: {{ template "app.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+  annotations:
+                                             kubernetes.io/ingress.class: "nginx"
```
Comment (mattops) [inline helm_deploy/community-proxy/templates/ingress.yaml]: formatting looks a bit out here

**Review by mattops — COMMENTED** [review]
Looks good.  One thing, do you want to add an IP whitelist?

**Review by mattops — DISMISSED** [review]

**Comment by andymarke** [comment]
> Looks good. One thing, do you want to add an IP whitelist?

@mattops I deliberately didn't add a whitelist as this service will only exist in dev/test as it currently stands. There is a small risk that a member of the public can call the health check endpoint since that URL is unprotected. 

If I were to add a whitelist do we have one where the service could called by 

1. Any developer in the office or at home
2. Any service in Azure
3. Any Service in AWS DPS (or whatever you call it)
4. Any Service in AWS CloudPlatform
5. Any Service in the new AWS Delius

If not it just might not be worth it given the primary aim of this is to help developers while we take forward the community_api post AWS migration

File: helm_deploy/community-proxy/templates/ingress.yaml (hunk 1 above)
Comment (andymarke) [reply helm_deploy/community-proxy/templates/ingress.yaml]: corrected formatting

**Review by petergphillips — APPROVED** [review]
LGTM

**Pull request merged**